Lib/fontTools/merge.py

```diff
@@ -788,7 +858,7 @@ def merge(self, fontfiles):
 			self._preMerge(font)
 
 		self.duplicateGlyphsPerFont = [{} for f in fonts]
-
+		self.fontfiles = fontfiles
```

Comment:
It is used in the warning message in GSUB merge, which outputs the font filename directly rather than out put its index.

Previously, the warning message looks like this:

glyphs xxx in font 3 is dropped. 

Now it reads:

glyphs xxx in NotoSans-Regular.ttf is dropped.

Which is more user friendly I think.